science/parallel-netcdf/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               mpi 1.0
+PortGroup               active_variants 1.1
```

Comment:
It looks like this line can be removed. I don't see the `active_variants` portgroup actually being used.